### PR TITLE
Remove "tagmail" report (not supported anymore)

### PR DESCRIPTION
**Pull request by njam**

### CODE REVIEW

**Comment by njam** [comment]
@cargomedia/devops fyi

There's no more "tagmail" (email) report in puppet. Fortunately we have reporting+alerting with bipbip.

**Comment by ppp0** [comment]
@njam I notice you do the same here in the `exec` as I'm doing in https://github.com/cargomedia/puppet-packages/pull/1671

Is/Was there a reason to use `exec` instead of managing the directory?

File: modules/puppet/manifests/agent.pp [hunk 1]
```diff
@@ -25,6 +25,13 @@
       notify  => Exec['/etc/puppetlabs/puppet/puppet.conf'];
   }
 
+  exec { 'chmod 0755 /opt/puppetlabs/puppet/cache':
```
Comment (ppp0) [inline modules/puppet/manifests/agent.pp]: @njam sorry, this is the code I was meaning in https://github.com/cargomedia/puppet-packages/pull/1278#issuecomment-281123958

Comment (njam) [inline modules/puppet/manifests/agent.pp]: hm sorry, don't remember - and commit message doesn't explain it

**Pull request merged**